fdbclient/ServerKnobs.cpp

```diff
@@ -355,6 +355,8 @@ void ServerKnobs::initialize(Randomize randomize, ClientKnobs* clientKnobs, IsSi
 	init( ROCKSDB_FETCH_QUEUE_HARD_MAX,                          100 );
 	init( ROCKSDB_FETCH_QUEUE_SOFT_MAX,                           50 );
 	init( ROCKSDB_HISTOGRAMS_SAMPLE_RATE,                      0.001 ); if( randomize && BUGGIFY ) ROCKSDB_HISTOGRAMS_SAMPLE_RATE = 0;
+	// Set to 0 to disable rocksdb write rate limiting.
+	init( ROCKSDB_WRITE_RATE_LIMITER_BYTES,                        0 ); if( randomize && BUGGIFY ) ROCKSDB_WRITE_RATE_LIMITER_BYTES = 0;
```

Comment:
I assume you would set to non-zero value for tests.

Reply:
Will decide and enable it, once I find the knob value which makes sense for simulation. Currently leaving it at 0(disabled)